src/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -642,6 +685,10 @@ std::string CodegenNeuronCppVisitor::get_variable_name(const std::string& name,
         // not printing a NET_RECEIVE block.
     }
 
+    if (stringutils::starts_with(name, "_ptable_")) {
```

Comment:
Just for my own understading: I was staring at this a bit to understand if the "hardcoding" is relevant: Is this for VERBATIM block? Or something else?

Reply:
Right, this is part is annoying. If you look at the test we can use FUNCTION_TABLE without a section. However, for historical reasons the global variables are part of the Instance struct. No section, no instance and hence no valid instance struct.

Therefore to avoid segfaults, we need to use the static variable `global_struct_instance()`; and can't simply rely on the usual way of translating a global variable into their C++ implementation.

I'm not aware of a different way of recognizing the global variables `_ptable_{}` for FUNCTION_TABLES.